Core/EventHandlers.lua

```diff
@@ -2082,5 +2088,22 @@ function Rarity:OnDisgustingVatFished()
 	addAttemptForItem("Emmah", "pets")
 end
 
+local dreamseedMounts = {
+	[L["Reins of the Winter Night Dreamsaber"]] = true,
+	[L["Reins of the Snowfluff Dreamtalon"]] = true,
+	[L["Reins of the Evening Sun Dreamsaber"]] = true,
+	[L["Reins of the Blossoming Dreamstag"]] = true,
+	[L["Reins of the Springtide Dreamtalon"]] = true,
+	[L["Reins of the Morning Flourish Dreamsaber"]] = true,
+	[L["Reins of the Rekindled Dreamstag"]] = true,
+}
+
+function Rarity:OnDreamseedCacheOpened()
+	Rarity:Debug("Adding attempts for " .. tostring(dreamseedMounts))
```

Comment:
That `tostring` will print something like `table: 0x7f9dc7197800`. You probably meant to move this into the loop and print the individual item names?

Reply:
i was actually going to remove that but had forgotten, should i leave it in?

Reply:
I'll remove it when rebasing, no problem.